### PR TITLE
Update target sdk and error screen

### PR DESCRIPTION
**Pull request by russellcullen**
**What's this do?**
Update target and compile sdk to prep for Google policy changes. In addition, this also updates the error reporting screen since the current "Report Issue" button sends the report to a generic, untracked email, we're replacing that button with some text to direct the user to their local library help email.

### CODE REVIEW

**Review by samanthaandrews — APPROVED** [review]
Looks good to me, but an Android dev should probably give final approval 👍

**Review by samanthaandrews — APPROVED** [review]

File: simplified-cardcreator/src/main/java/org/nypl/simplified/cardcreator/ui/LocationFragment.kt [hunk 1]
```diff
@@ -292,7 +292,7 @@ class LocationFragment : Fragment(), LocationListener {
     val address: Address?
 
     try {
-      address = geocoder.getFromLocation(location.latitude, location.longitude, maxResults)[0]
+      address = geocoder.getFromLocation(location.latitude, location.longitude, maxResults)!![0]
```
Comment (russellcullen) [inline simplified-cardcreator/src/main/java/org/nypl/simplified/cardcreator/ui/LocationFragment.kt:295]: both these force unwraps are contained in generic `try {} catch (e: Exception) {}` blocks so it's 'safe' to do. (as safe as if we had a null address here, the exception is caught either way)

File: simplified-cardcreator/src/main/java/org/nypl/simplified/cardcreator/ui/LocationFragment.kt [hunk 2]
```diff
@@ -230,7 +230,7 @@ class LocationFragment : Fragment(), LocationListener {
 
     try {
       if (location != null) {
-        address = geocoder.getFromLocation(location.latitude, location.longitude, maxResults)[0]
+        address = geocoder.getFromLocation(location.latitude, location.longitude, maxResults)!![0]
```
Comment (MalcolmMcFly) [inline simplified-cardcreator/src/main/java/org/nypl/simplified/cardcreator/ui/LocationFragment.kt:233]: Might be more readable like `address = geocoder.getFromLocation(location.latitude, location.longitude, maxResults).first()`

File: simplified-cardcreator/src/main/java/org/nypl/simplified/cardcreator/ui/LocationFragment.kt (hunk 1 above)
Comment (MalcolmMcFly) [inline simplified-cardcreator/src/main/java/org/nypl/simplified/cardcreator/ui/LocationFragment.kt:295]: Same as above, might be more readable like `address = geocoder.getFromLocation(location.latitude, location.longitude, maxResults).first()`

File: simplified-cardcreator/src/main/java/org/nypl/simplified/cardcreator/ui/LocationFragment.kt [hunk 3]
```diff
@@ -107,7 +107,7 @@ class LocationFragment : Fragment(), LocationListener {
       if (isNewYork || locationMock != null) {
         locationMock?.let { location ->
           activityModel.userLocationAddress =
-            geocoder.getFromLocation(location.latitude, location.longitude, 1)[0]
+            geocoder.getFromLocation(location.latitude, location.longitude, 1)?.get(0)
```
Comment (MalcolmMcFly) [inline simplified-cardcreator/src/main/java/org/nypl/simplified/cardcreator/ui/LocationFragment.kt:110]: Might be more readable like: `geocoder.getFromLocation(location.latitude, location.longitude, 1).first()`

**Review by MalcolmMcFly — APPROVED** [review]
Very minor non-blocking comments. This code probably won't be supported long term so take my suggestions here with a grain of salt.

File: simplified-cardcreator/src/main/java/org/nypl/simplified/cardcreator/ui/LocationFragment.kt (hunk 1 above)
Comment (MalcolmMcFly) [reply simplified-cardcreator/src/main/java/org/nypl/simplified/cardcreator/ui/LocationFragment.kt:295]: No biggie, just aiming for readability. Good to merge 👏

**Pull request merged**